include/pdflib.h

```diff
@@ -1326,6 +1342,13 @@ struct PDFlib_api_s {
                 const char *keyword, const char *optlist);
     void (PDFLIB_CALL * PDF_initgraphics)(PDF *p);
     void (PDFLIB_CALL * PDF_lineto)(PDF *p, double x, double y);
+    int (PDFLIB_CALL * PDF_load_graphics)(PDF *p, const char *type, const char *filename,
+                int len, const char *optlist);
+    void (PDFLIB_CALL * PDF_close_graphics)(PDF *p, int graphics);
+    void (PDFLIB_CALL * PDF_fit_graphics)(PDF *p, int graphics, double x, double y,
+                const char *optlist);
+    double (PDFLIB_CALL * PDF_info_graphics)(PDF *p, int graphics, const char *keyword,
+                const char *optlist);
```

Comment:
Could these method definitions be in alphabetical order, among all the other method definitions?